_includes/docs/latest/deployment_prerequisites.md

```diff
@@ -76,6 +76,15 @@ Also, you can grant the roles on the organization, folder, or project IAM polici
     $ gcloud projects add-iam-policy-binding PROJECT_ID \
        --member=MEMBER_TYPE:MEMBER_NAME --role=ROLE_NAME
     ```
+    
+  * Service Account IAM:
+
+    ```bash
+    gcloud iam service-accounts add-iam-policy-binding YOUR_SERVICE_ACCOUNT \
+      --member=serviceACcount:YOUR_SERVICE_ACCOUNT \
+      --role=ROLE_NAME
+    ```
+
 
 The `MEMBER_TYPE` value is either `user`, `group`, `serviceAccount`, or `domain`.
```

Comment:
This section needs updated to the variable configuration format established earlier.

Reply:
done